Remove Lorem Ipsum and put in first draft of words

thk123 opened the pull request.

thk123 committed:
Remove Lorem Ipsum and put in first draft of words

thk123 commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 
+ Each of the talks features practical examples using Python, [Git](https://git-scm.com/), Gitlab. After each talk I will answer questions - giving plenty of time for the specific challenges you are having using these best practises. I also like to foster an on-going relationship after the talks to help you put these tools and techniques into practise.
 
 ### Souce Control
```
Possibly move all of these sections to a syllabus page, then add a course leader section

kbarrett replied:
Typo: s/Souce/Source

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 
```
Serif font for contact details because it's in the title?

kbarrett replied:
I still think this.

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 **If your company has domain experts who are writing code, [get in touch](mailto:hi@tkiley.co.uk) today to increase both the reliability of the software you produce, as well as the speed at which you can develop it.**
 
-You can read more about what the course covers and how it is structured [here](course.html). 
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt the importance of good software practises like writing automated tests to check the code is correct and using continuous integration.
```
s/like writing automated tests to check the code is correct and using continuous integration//

Unnecessary technical details in the intro.

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt the importance of good software practises like writing automated tests to check the code is correct and using continuous integration.
 
-Get in touch today to discuss getting these seminars given to you. Email me [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) or ring me on 07948 284729.
+Writing code is now an essential part of all science and technology research and development - yet many of the scientists and domain experts have never received training on software engineering best practices, and are instead expected to figure it out. The result of this is, at best, slow software development, and at worst incorrect research results.
```
❤️

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+**Get a tailored seminar series that will meet the specific needs of your company.**
+
+The seminars themselves can be tailored to the specific needs of your company. I am also available following the talks to have group or one-on-one meetings to discuss the specific software engineering challenges you face, and work with you to come up with a solution that works for everyone.
```
Intro the seminars before referencing them? "I can deliver 4 hour-long seminars..."

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+The seminars themselves can be tailored to the specific needs of your company. I am also available following the talks to have group or one-on-one meetings to discuss the specific software engineering challenges you face, and work with you to come up with a solution that works for everyone.
+
+You can read more about what the course covers and how it is structured [here](#the-course).
```
✂️ It's weird to skip forward over the bit you want them to read!

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 ## The Course
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+The course is designed to cover the key areas of modern software engineering best practises. Each module will help developers to adopt industry standard approaches to developing software. These cover both technical topics such as writing automated tests, as well as procedural topics such as en effective code review process.
```
Consistent language? Course vs seminars.

kbarrett replied:
Typo: s/en/an/

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+The course is designed to cover the key areas of modern software engineering best practises. Each module will help developers to adopt industry standard approaches to developing software. These cover both technical topics such as writing automated tests, as well as procedural topics such as en effective code review process.
+
+You may already be using some of these techniques, so the course is entirely customsisable to exclude the parts you already do, or provide more details on what is mission critical to you.
```
Typo: s/customsisable/customisable/

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
- 5. [Continuous Integration (CI)](#continuous-integration-ci) 
+ 5. [Continuous Integration (CI)](#continuous-integration-ci)
 
+ Each of the talks features practical examples using Python, [Git](https://git-scm.com/), Gitlab. After each talk I will answer questions - giving plenty of time for the specific challenges you are having using these best practises. I also like to foster an on-going relationship after the talks to help you put these tools and techniques into practise.
```
Consistent language: modules vs talks

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 ### Souce Control
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+Source control is a tool that stores your source code, and tracks changes to that code across time. Popular source control software includes [Git](https://git-scm.com/) and [Subversion](https://subversion.apache.org/).
```
More focus on why they would want to learn/use source control rather than what it is?

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+Source control is a tool that stores your source code, and tracks changes to that code across time. Popular source control software includes [Git](https://git-scm.com/) and [Subversion](https://subversion.apache.org/).
+
+If you are not already using source control, this module will teach you how to use Git - the most popular source control amongst professional developers to manage changes to your source control, including using task branches (the "Github flow" workflow) and resolving merge conflicts.
```
Ditto - focus on why not what. e.g. instead of "resolving merge conflicts" talk about making it easier for multiple people to work on the same project.

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+If you are not already using source control, this module will teach you how to use Git - the most popular source control amongst professional developers to manage changes to your source control, including using task branches (the "Github flow" workflow) and resolving merge conflicts.
+
+**Source control is vital for working with other developers on code. It allows for fast iteration, whilst maintaining stability of the software. Learning to use source control will revolutionise your software development process.**
```
❤️

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+**Source control is vital for working with other developers on code. It allows for fast iteration, whilst maintaining stability of the software. Learning to use source control will revolutionise your software development process.**
+
+If you already use source control, well done! This module can be tailored to whatever tools you use and to cover best practises for using them effectivey. Get in touch today to discuss how to make this module be useful for your company.
```
I like the existence of this bit, but not sure about the tone.

thk123 committed:
Merge remote-tracking branch 'origin/main' into update-content

thk123 commented:
Artifact: https://github.com/thk123/se4be-website/suites/10331334646/artifacts/508583832

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+This module will explain all the ways in which automated tests are indispensible. It will cover how to write tests, even when the code is complicated, with plenty of Python examples.
+
+**In my experience I have seen that code that isn't tested does not work as the writer intends. Learning to write tests will help your software developement process be more correct in the long term, and faster in the short term.**
```
Typo: s/developement/development

(Maybe it's easier to set up a spell-check rather than me doing this!)

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 ### Code Review
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+A code review process involves having one developer review another developers changes to the source code before it is submitted. It is a key step to ensuring that code is readable and tested. It also serves to reduce the "bus factor" of the project. It also upskills developers on code writing techniques for free.
```
"It is a key step to ensuring that code is readable and tested. It also serves to reduce the "bus factor" of the project. It also upskills developers on code writing techniques for free." -> "It is a key step to ensuring that code is understood by everyone."

Less technical detail, e.g. "bus factor" might not be understood by everyone.

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
+
+**Code is a communication tool, writing good code will minimize misunderstanding between past and future developers.**
+
+This module can be adjusted by reviewing some of your existing code to see the kind of problems that your experiencing and focusing on the techniques that will apply to your code base. Get in touch today to start learning how to write better code.
```
Links every time you say "get in touch today"?

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 ### Continuous Integration (CI)
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+Using Continuous Integration technologies like Gitlab Actions or CircleCI is a very cheap and easy way to increase the reliability of your software. In conjunction with source control and automated tests, every change to the code is checked to actually do what it is supposed to do. This frees developers up from testing changes by hand and to focus on their important work.
```
Again focus on the benefit of using this tech rather than what it does.

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
\ No newline at end of file
+Each of the modules cost £1200 to deliver. An on-going consulting after the talks is £600 per day, or £100 per hour.
```
I'd be tempted to make this a section about the service you're providing without specific prices. But mentioning that you provide consulting as well is great!

kbarrett commented on index.md:
```diff
@@ -8,45 +8,99 @@ I offer a series of seminars designed to help scientists, engineers and other do
\ No newline at end of file
+Each of the modules cost £1200 to deliver. An on-going consulting after the talks is £600 per day, or £100 per hour.
+
+I am deeply passionate about helping domain experts become better software engineers. If this pricing structure doesn't work for you, or you want to tailor the course to your business needs then please get in touch and we can discuss a pricing structure that works for you.
```
I like this bit: "to tailor the course to your business needs then please get in touch and we can discuss a pricing structure that works for you."

kbarrett reviewed (commented):
Sorry that it's a bit solutions rather than problems but I tried to explain why I was suggesting it.

thk123 commented:
Actions:
 - [x] setup a spell check in CI
 - [x] ensure consistent language: course / module / seminar
 - [x] Each module section should have a non-techy paragraph, and a section covering technologies covered
 - [x] Introduce the whole concept in advance of anything about customisation

thk123 committed:
Merge remote-tracking branch 'origin/main' into update-content

thk123 committed:
Update ignored words

Latin words no longer required - adding in some technical / trademarked
words

thk123 committed:
Adjust pipeline to convert markdown into html before doing the spell check

thk123 committed:
Fix spelling mistakes in existing content

thk123 committed:
Make course and seminar used consistently throughout

Opted to exclude module as was making some sentences overly clunky given
the 1-2-1 relationship between module and seminar.

thk123 committed:
Make opening paragraphs of each seminar description non-technical

Move the specific technologies to a specific section

thk123 committed:
Add option to emphasise key parts

thk123 committed:
Addressed comments and started highlighting things

thk123 committed:
Further tweaks to the words

thk123 committed:
Further tweaks to the words to make more punchy

thk123 committed:
Fix typos found in remote CI

thk123 committed:
More UK spellings

kbarrett commented on _sass/custom.scss:
```diff
@@ -8,6 +8,12 @@ h1, h2, h3 {
     font-family: Helvetica, Arial, sans-serif;
 }
 
+em {
```
I'm finding this hard to read.

kbarrett replied:
And confusing that it's the same colour as links.

thk123 replied:
If possible I'd like to address this interactively, as I am a bit close to see what looks good

thk123 replied:
Also noting neither of my reference sites really do this

kbarrett commented on index.md:
```diff
@@ -4,49 +4,151 @@ title: Software Engineering for Busy Experts
+I can deliver a course that will teach your developers the _industry standard techniques_ to write more reliable code faster.
 
-> "Having Thomas come on site and give these talks massively improved our productivity"
+The course consists of _5 seminars_, with lots of time for questions. After the course, I would provide an _on-going relationship_ to help you apply the techniques.
```
This doesn't feel like the most important thing to have at the top of the page. Ditto the next line.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,151 @@ title: Software Engineering for Busy Experts
+
+Get in touch today to discuss receiving this course. Email me [hi@tkiley.co.uk](mailto:hi@tkiley.co.uk) to arrange a call to discuss booking in this course.
+
+## About Thomas
```
The information hierarchy seems a little muddled now. Think about which bits are the most important for someone to see when they come to your page.

I'd swap this & "the course" over (but not the details about the curriculum) and move the 5 seminars bit above to the (now separate) section about the curriculum.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,151 @@ title: Software Engineering for Busy Experts
+
+#### Topics & Technologies
+
+The off-the-shelf seminar will cover:
```
"This seminar can cover:" ??

kbarrett reviewed (commented):
The words I've read so far seem good but I didn't get all the way through. I think the sections need rearranging though.

thk123 committed:
Make initial call to action more punchy

thk123 committed:
Reorder paragraphs to priorize info

People want to know what the course consists of at a high level before
they know who I am, but who I am is the next priority.

thk123 committed:
Simplify sentence structure

thk123 committed:
Simplify sentence

thk123 committed:
Make titles not have prefix

thk123 committed:
Make the syllabus for each seminar consistent

thk123 committed:
Make services section fit the format

thk123 committed:
Make each seminar section have just one paragraph before the punch quote

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+{% include opening_box.html content="I offer a course designed to help scientists, engineers and other domain experts to learn the most important software engineering best practices to help them write correct code more efficiently." %}
 
-**If your company has domain experts who are writing code, [get in touch](mailto:hi@tkiley.co.uk) today to increase both the reliability of the software you produce, as well as the speed at which you can develop it.**
+**[Get in touch](mailto:hi@tkiley.co.uk) today to learn how to use software industries best practices at your work.**
```
"at your work" is quite colloquial.

s/at/in

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+> *A Company*
 
-> "Having Thomas come on site and give these talks massively improved our productivity"
+I can deliver a course that will teach your developers the _industry standard techniques_ to write more reliable code faster.
```
If you decide to keep this, add the consulting? e.g. "a course and on-going support"

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt _the importance of good software practices_ in writing correct code quickly, and in collaboration with others.
+
+In my time as team lead I was able to see how good software practices can _speed up delivery_ and increased quality.
```
"In my time as team lead I was able to see" -> "I have seen first-hand how"

So it reads less like a CV.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
- 5. [Continuous Integration (CI)](#continuous-integration-ci)
+ 5. [Continuous Integration](#continuous-integration)
+
+Each of the seminars features _practical examples_ to see how these techniques can be applied. These seminars take the form of a 45 minute presentation and time for questions. This interactivity ensures the techniques can be applied in your domain.
```
"and time for questions" -> "plus time for questions"

I think this makes it clearer that the question time isn't included in the 45 minutes.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+The seminars can be delivered in person at your office, via a video call or a hybrid approach.
+
+In addition to the off-the-shelf course, I am happy to tailor the course to _match your needs, experience and technologies_.
```
Remove "In addition to the off-the-shelf course,"

I think referencing the base course explicitly makes it _sound_ less custom.

thk123 replied:
Remove all references to off-the-shelf

thk123 replied:
Q: what about the last reference?

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 ## The Course
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+The course is designed to cover the key areas of modern software engineering best practices. Each seminar will help developers to adopt _industry standard approaches_ to developing software.
```
💡 Remove this section and add more detail to the "I can deliver a course" paragraph above with any detail you want to keep. Then I think it's less weird that there's not as much detail and the opening section is essentially all about the course and why they want it anyway.

thk123 replied:
Need to smoosh the two paragraphs to not repeat, then done

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+**[Get in touch](mailto:hi@tkiley.co.uk) today to craft a bespoke course for your business.**
+
+I also like to foster an on-going relationship after you have completed the course to _help you put these tools and techniques into practice_.
```
Starting a section with "also" reads weird to me. Change to "I want to"?

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn the skills needed for _effective software development_.
 
+### Source Control
```
"Seminar 1: Source Control"

I'd forgotten that we were in a specific context by the time we get to this heading so add a reminder?

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+### Source Control
 
-### Souce Control
+Source control is a tool that stores your source code, and tracks changes made to it across time. It facilitates collaboration between multiple developers working on the same code, as well as ensuring reproducible results.
```
Flip this - put why you would use it first.

"Source control is a tool that facilitates collaboration between multiple developers working on the same code, as well as ensuring reproducible results. It does this by storing your source code, and tracking changes made to it across time."

thk123 replied:
Go through and do for all of them

thk123 replied:
Done for all but C "good code" - which I will come back to

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+**Source control is indispensable for working with other developers on code.**
+
+If you already use source control then this seminar can be tailored to whatever tools you are already using. I will cover best practices for using them successfully.
```
Add what you will do if they're not already using it?

kbarrett replied:
You also kinda already say this later in the syllabus section, so maybe remove it - but I still think it's useful to say what you will teach in this module.

thk123 replied:
Replace this paragraph with something that explains the seminar

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+ * [Git-Cola](https://git-cola.github.io/){:target="_blank"} - a user friendly tool for interacting with Git
+ * [Kdiff3](https://kdiff3.sourceforge.net/){:target="_blank"} - a user friendly tool for resolving merge conflicts
+
+This seminar can be tweaked to use different source control software, hosting options to match your existing workflows.  Please get in touch with me and we can customise this talk to meet your needs and match your skills.
```
"different source control software, hosting options" <- Should the comma be an "and"? Otherwise I don't understand this sentence.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 ### Automated Tests
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+Automated tests automatically check code does what the developer intends. Learning to write automated tests will free the domain experts from having to constantly re-test changes by hand.
```
Again I think this reads better flipped. And I don't like the word "re-test".

"Learning to write automated tests will free the domain experts from having to constantly check their changes by hand. Instead, automated tests automatically check code does what the developer intends."

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+**Writing automated tests will eliminate new bugs and speed up collaboration.**
+
+Automated tests are one of the most powerful tools in a developers toolkit. My motto forged from years of practical experience is _code that isn't tested does not work_ as expected
```
Missing full-stop and it's weird that the emphasis doesn't include the whole quote.

s/does not work_ as expected/does not work as expected_.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+ * Why tests are valuable.
+ * How to write tests (including when it is tricky).
+ * When to write tests.
+ * Running tests
```
Either end with a full-stop or not, but be consistent (including across the other seminars).

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 ### "Good" Code
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+Writing code is relatively easy, but writing "good" code is
```
Not sure about this. Most people don't think writing code is easy.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+Writing code is relatively easy, but writing "good" code is
+elusive. By following industry approaches over time the quality of the code will improve.
```
"over time the quality of the code will improve" feels a bit of a rubbish goal - maybe pick something more flashy for your opener!

thk123 replied:
Think about making this a bit punchier - e.g. improve the code you write tomorrow, which will in turn improve your whole code base

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+2. Testable
+3. Maintainable
+
+This seminar will justify why writing good code is important, before providing a _large toolkit of guidelines and techniques_ that can be applied to your code to make it more readable, testable and maintainable.
```
The descriptions for these later seminars feel more fleshed out and complete than the earlier ones. That's probably better but at the very least it's strange that's there's such a difference.

thk123 replied:
Try and make all seminars have about the same amount of content

thk123 replied:
Maybe shrink this by getting rid of the ol

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+This seminar will justify why writing good code is important, before providing a _large toolkit of guidelines and techniques_ that can be applied to your code to make it more readable, testable and maintainable.
+
+This seminar can be adjusted by reviewing some of your existing code to see the kind of problems that your experiencing and focusing on the techniques that will apply to your code base. [Get in touch](mailto:hi@tkiley.co.uk) today to start learning how to write better code.
```
My inner grumpy journalist says: Passive voice.

"I can review your existing code to focus on the techniques that will help you to fix the kind of problems that you're experiencing in your code base." (or something similar)

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+ * How to make code testable.
+ * How to make code maintainable.
+
+ Software & Technologies:
```
Change this heading to "Example software & technologies" to make it clearer what they are. Or "Relevant software & technologies"?

thk123 replied:
S&T Used / Or Example S&T

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
 ### Code Review
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectu ...
+A code review process involves having one developer review another developers changes to the source code before it is submitted. Doing this effectively can spot defects and share knowledge.
```
Flip to put the "why" bit first (and missing apostrophe):

"A code review process can spot defects and share knowledge. It involves having one developer review another developer's changes to the source code before it is submitted."

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+**Having a code review process will simultaneously upskill your developers and improve the quality of your code.**
+
+Code review _ensures that the code understood by everyone_, as well as sharing coding best practices organically.
```
Missing "is" between "code" & "understood".

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+ * [Gitlab](https://about.gitlab.com/){:target="_blank"} - a widely used cloud tool for conducting code reviews
+
+As with all the other seminars, get in touch if you use a different platform for conducting code reviews and I can tailor the course to use that website to make it easier to apply this to your existing workflows.
```
Add "want" to make it relevent to everyone? e.g. "if you want to use a different platform"

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+### Continuous Integration
+
+Continuous Integration (CI) is an industry standard approach to automatically check changes to projects automatically. This catches problems before they affect other developers and customers.
```
Flip it 😝

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+**Continuous Integration allows developers to make changes confidently and efficiently.**
+
+Using Continuous Integration (CI) technologies is a cheap and easy way to increase the reliability of your software. Every change to the code is checked automatically, freeing experts up from testing changes by hand and to focus on their expertise. It helps developers work together, without breaking each others work.
```
Defining acronyms multiple times is confusing.

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+## Services
+
+I offer an off-the-shelf course, with the option to tailor any and all of the seminars to better meet your precise needs and technologies.
```
Again I wouldn't mention "off-the-shelf". e.g. "I offer a course covering the most important aspects of software development best practices, with the option..."

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+I offer an off-the-shelf course, with the option to tailor any and all of the seminars to better meet your precise needs and technologies.
+
+**[Get in touch](mailto:hi@tkiley.co.uk) today to discuss getting this course at your business.**
```
This is a bit colloquial. Suggested alternative: "to discuss how your business can benefit from this course."

kbarrett commented on index.md:
```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+
+**[Get in touch](mailto:hi@tkiley.co.uk) today to discuss getting this course at your business.**
+
+The seminars can be delivered at on-site, via a zoom call or a hybrid approach - whichever works best for you.  I am based in Oxford, UK.
```
s/zoom/Zoom

thk123 replied:
Better - video call

kbarrett reviewed (commented):
Looking great! Just a couple of suggestions - and think I've got a solution to your "course" section.

thk123 committed:
Give the section names seminar prefix

This emphasises what they are. Also made the bullet point use the same
system for enumerating them

thk123 committed:
Addressing various review comments on the content

thk123 committed:
Make first paragraph of the 5 semianrs be consistent

thk123 committed:
Make each seminar section has consistent amount of content

thk123 committed:
Ensure all seminar sections have contact details

thk123 committed:
Make pull quotes consistent

thk123 committed:
Ensure all syallbi have a consistent call to action and mention customising

thk123 committed:
Fix double spacing issue

thk123 committed:
Fix typos

thk123 committed:
Add word spell checker doesn't know
